src/sql/azdata.d.ts

```diff
@@ -3198,6 +3198,10 @@ declare module 'azdata' {
 	export interface TextColumnOption {
 	}
 
+	export interface ButtonColumnOption {
+		iconClass?: string;
```

Comment:
please refer to other places where extensions need to provide icons, it should be the: string | {light:string, dark:string}, the string is the path to the icon, and we will generate css rules in the ADS core for the provided icons, it is not feasible for the extension to provide a class name